data-manipulation/encryption/rsa/rsa-encrypt.yml

```diff
@@ -0,0 +1,45 @@
+rule:
+  meta:
+    name: rsa encrypt
```

Comment:
Let's update this rule name to _encrypt data using RSA via YY_. If we don't have a good substitute for _YY_ then we should combine _encrypt data using RSA via WinAPI_ (below) with this rule under the name _encrypt data using RSA_.

Reply:
I think it is better to keep them separated to make the rules easy to understand. I am not completely sure what a good _YY_ would be, what about _encrypt data using RSA via embedded library`? @mandiant/capa-dev any other ideas?

Reply:
My first thought is _...via x86 assembly_ but with the calls included maybe _...via static library_ is a better fit?